docs/aerodromes/classc/Brisbane.md

```diff
@@ -112,15 +112,88 @@ Non-Jet aircraft, and aircraft that cannot accept a Procedural SID, shall be ass
+
+### Waypoints
+In VMC, all helicopters shall be processed via two visual waypoints (outlined below) positioned north and south of the field. Where VMC does not exist, IFR helicopters will conform to fixed wing ops and be processed via the **BN (RADAR) SID** from an appropriate runway.
+
+
+| Direction of Flight  | Inbound/Outbound Tracking Point |
+| ------------------ | -------------- |
+| North      | BLHS   |
+| South      | MBHM   |
+| East      | MBHM   |
+| West      | BLHS   |
+
+### Departures
+Where a helicopter has not filed via the appropriate waypoint, they should be rerouted.
+
+!!! example
+    **RSCU533:** "Brisbane Delivery, helicopter RSCU533, MEDEVAC, for YXHE, request clearance"  
+    **BN ACD:** "RSCU533, Brisbane Delivery, cleared to YXHE via amended BLHS, YXHE, climb A025, squawk 1256, departure frequency 124.7"  
+    **RSCU533:** "Cleared to YXHE via amended BLHS, YXHE, climb A025, squawk 1256, departures 124.7, RSCU533"
+
+Both taxiway **H2** and **F4** are inside the maneuvering area and treated like a runway, requiring a takeoff and landing clearance from BN ADC. Helicopters departing on other surfaces outside the maneuvering areas should be instructed to *"report airborne"*. 
+
+!!! example
+    **RSCU533:** "Brisbane Tower, helicopter RSCU533, H2 ready"  
+    **BN ADC:** "RSCU533, Brisbane Tower, H2 cleared for takeoff"  
+    **RSCU533:** "H2 cleared for takeoff, RSCU533"
+ 
+    **BN ADC:** "RSCU533, contact departures"  
+    **RSCU533:** "Departures, RSCU533"
+
+### Arrivals
+Helicopters arriving to YBBN during VMC shall be cleared by **BN TCU** via the appropriate inbound waypoint, with that waypoint issued as the clearance limit. TCU shall coordinate the arrival with ADC and transfer the aircraft approaching the clearance limit.
+
+ADC shall assess the current traffic situation and, if necessary, instruct the pilot to hold and/or maintain own separation with any arriving or departing aircraft, then clear them for a visual approach to their nominated landing site. 
+
+If holding is required prior to arrival, helicopters shall be instructed to hold at the following positions:
+
+| Inbound Via | Holding Position |
+| ----------- | ---------------- |
+| BLHS        | Brisbane Entertainment Centre *(3nm northwest of runway 01L/19R)* |
+| MBHM        | MBHM             |
+
+!!! phraseology
+    <span class="hotline">**TCU** -> **ADC**</span>: "North, helicopter X6G, inbound at A010, clearance limit BLHS"  
+    <span class="hotline">**ADC** -> **TCU**</span>: "X6G"  
+
+    **X6G**: "Brisbane Tower, helicopter X6G, for taxiway F4"  
+    **BN ADC**: "X6G, Brisbane Tower, cancel clearance limit, track to and hold at the Brisbane Entertainment Centre, maintain A010"  
+    **X6G**: "Cancel clearance limit, track to and hold at the Brisbane Entertainment Centre, maintain A010, X6G"
+
+    **BN ADC**: "X6G, report sighting a Qantas 737, 4nm final runway 19R"  
```

Comment:
```suggestion
    **BN ADC**: "X6G, report sighting, and able to maintain own separation with, a Qantas 737, 4nm final runway 19R"  
```

By the book, this extra step is required. IRL, it is usually omitted for operators that use this standard regularly